cmd/snap/cmd_debug_state.go

```diff
@@ -95,17 +95,22 @@ func (t byLaneAndWaitTaskChain) Swap(i, j int) { t[i], t[j] = t[j], t[i] }
 func (t byLaneAndWaitTaskChain) Less(i, j int) bool {
 	// cover the typical case (just one lane), and order by first lane
 	if t[i].Lanes()[0] == t[j].Lanes()[0] {
```

Comment:
given that this is a less predicate for paranoia, given we are changing this, we should also add here:

```
if t[i].ID() == t[j].ID() {
   return false
}
```

Reply:
Done